src/wagmiConfig/index.ts

```diff
@@ -226,3 +226,6 @@ export const wagmiConfig = createConfig({
         }),
       ],
 })
+
+export const COINBASE_INJECTED_WALLET_ID = "com.coinbase.wallet"
+export const COINBASE_WALLET_SDK_ID = "coinbaseWalletSDK"
```

Comment:
Extract these constants into a `constants.ts` file.

Reply:
Yeah, good suggestion & we should probably make a `config/wagmi/index.ts`, `config/wagmi/constants.ts` & `config/fuel/index.ts` files, but I wouldn't like to change the directory structure in this PR, so let's do it separately.